app/models/services/cbd_commitment_hash.rb

```diff
@@ -23,6 +27,21 @@ def call
     }
   end
 
+  def cfn_approved
+    return false if has_government_managers?
+
+    @commitment.cfn_approved
```

Comment:
Could you simplify this to `!has_government_managers? && @commitment.cfn_approved`?

Reply:
i don't think that simplifies it. it does give the correct answer, but it's harder to figure out what the answer is just by looking at it. i had to check that it returns the nil